cli/src/commands/invocations/describe.rs

```diff
@@ -100,14 +100,7 @@ async fn describe(env: &CliEnv, opts: &Describe) -> Result<()> {
 
     c_title!("🚂", "Invocation Progress");
     if let Some(parent) = &inv.invoked_by_id {
-        c_println!(
-            "{} {}",
-            inv.invoked_by_component
-                .as_ref()
-                .map(|x| style(x.to_owned()).italic().blue())
-                .unwrap_or_else(|| style("<UNKNOWN>".to_owned()).red()),
-            style(parent).italic(),
-        );
+        c_println!("{}", style(parent).italic());
```

Comment:
If we really need this info back in the CLI, we could reintroduce it later with a join in the query.

Reply:
It was really useful, if anything, we want to improve observability of invocation trees.